Add SPC5 to generic config updater file

noaOrMlnx opened the pull request:
<!--
    Please make sure you've read and understood our contributing guidelines:
    https://github.com/Azure/SONiC/blob/gh-pages/CONTRIBUTING.md

    ** Make sure all your commits include a signature generated with `git commit -s` **

    If this is a bug fix, make sure your description includes "closes #xxxx",
    "fixes #xxxx" or "resolves #xxxx" so that GitHub automatically closes the related
    issue when the PR is merged.

    If you are adding/modifying/removing any command or utility script, please also
    make sure to add/modify/remove any unit tests from the tests
    directory as appropriate.

    If you are modifying or removing an existing 'show', 'config' or 'sonic-clear'
    subcommand, or you are adding a new subcommand, please make sure you also
    update the Command Line Reference Guide (doc/Command-Reference.md) to reflect
    your changes.

    Please provide the following information:
-->

#### What I did
Add spc5 to generic_config_updater file
#### How I did it

#### How to verify it

#### Previous command output (if the output of a command-line utility has changed)

#### New command output (if the output of a command-line utility has changed)

noaOrMlnx committed:
Add SPC5 to generic config updater file

stephenxs commented on generic_config_updater/gcu_field_operation_validators.conf.json:
```diff
@@ -20,9 +20,12 @@
                 "spc1": [ "ACS-MSN2700", "ACS-MSN2740", "ACS-MSN2100", "ACS-MSN2410", "ACS-MSN2010", "Mellanox-SN2700", "Mellanox-SN2700-C28D8", "Mellanox-SN2700-D40C8S8", "Mellanox-SN2700-D44C10", "Mellanox-SN2700-D48C8",
                           "ACS-MSN2700-A1", "Mellanox-SN2700-A1", "Mellanox-SN2700-A1-C28D8", "Mellanox-SN2700-A1-D40C8S8", "Mellanox-SN2700-A1-D44C10", "Mellanox-SN2700-A1-D48C8" ],
                 "spc2": [ "ACS-MSN3800", "Mellanox-SN3800-D112C8", "ACS-MSN3420", "ACS-MSN3700C", "ACS-MSN3700", "Mellanox-SN3800-C64", "Mellanox-SN3800-D100C12S2", "Mellanox-SN3800-D24C52", "Mellanox-SN3800-D28C49S1", "Mellanox-SN3800-D28C50" ],
-                "spc3": [ "ACS-MSN4700", "ACS-MSN4600", "ACS-MSN4600C", "ACS-MSN4410", "ACS-SN4280", "Mellanox-SN4600C-D112C8", "Mellanox-SN4600C-C64", "Mellanox-SN4700-O8C48", "Mellanox-SN4600C-D100C12S2", "Mellanox-SN4600C-D48C40",
```
`ACS-SN4280` is removed. by mistake?
the rest LGTM

noaOrMlnx replied:
yes will fix

noaOrMlnx replied:
rebase issue - fixed

BYGX-wcr committed:
Enable show ip bgp on sup and -n all for show ip bgp network (#3417)

#### What I did

1. Enable "show ip bgp" on sup and "-n all" for show ip bgp network.
2. Modify README.doc to make the instructions of building and installing the wheel package more clear.
3. Improve the output format of rexec command

#### How I did it

Modify the code in show/main.py to enable "show ip bgp ..." on supervisors and modify show/bgp_frr_v4.py to add support for the new features.
Update README.md.
Modify the rexec implementation to improve the output format.
Add unit tests for the above change.

#### How to verify it

Run on a SONiC chassis

mihirpat1 committed:
[SfpUtil] sfp eeprom with option dom is not working on Xcvrs with flat memory (#3385)

Signed-off-by: Mihir Patel <patelmi@microsoft.com>

ryanzhu706 committed:
Update DB version to 202411 on master branch. (#3414)

* Update DB version to 202411 on master branch.

ganglyu committed:
fix show techsupport date issue (#3437)

What I did
Show techsupport is designed to collect logs and core files since given date.
I find that some core files are missing when given date is relative, for example "5 minutes ago".
Microsoft ADO: 28737486

How I did it
Create the reference file at the start of the script, and don't update it in find_files.

How to verify it
Run end to end test: show_techsupport/test_auto_techsupport.py

ridahanif96 committed:
IP Assignment Issue (#3408)

#### What I did

Added Check for IP Assignment on Port when a Vlan is configured.
This PR is created in response to [Issue](https://github.com/sonic-net/sonic-buildimage/issues/19505)

#### How I did it

Modified config/main.py to add check for IP Assignment when Port has vlan membership 
#### How to verify it

After this, ip cannot be assigned on port which is configured to a VLAN.

bktsim-arista committed:
Fix multi-asic behaviour for dropstat (#3059)

* Fixes dropstat multi-asic behaviour by using multi-asic helpers
and ensuring that dropstat iterates through correct namespaces
when 'show' command is run.


Co-authored-by: rdjeric <rdjeric@arista.com>
Co-authored-by: Kenneth Cheung <kennethcheung@arista.com>

AnoopKamath committed:
Add sfputil power enable/disable command (#3418)

BYGX-wcr committed:
Change the default behavior of show ip bgp network (#3447)

* show/bgp_frr_v4.py: change the default behavior of "show ip bgp network"

- after change, show ip bgp network will have "all" as the default value of namespace option
- after change, ip-address/ip-prefix is a required argument when executing show ip bgp network on a chassis supervisor

* tests/remote_show_test.py update unit tests to comply with the new behaviors

* tests/show_bgp_network_test.py: update a test vector to make it comply with the new default behavior

* tests/bgp_commands_input/bgp_network_test_vector.py: update a test vector to comply with the new default behavior

xincunli-sonic committed:
Add namespace check for multiasic (#3458)

* Add namespace check for multiasic

* Fix format

BYGX-wcr committed:
Update README.md (#3406)

* Update README.md

The new location of the sonic-utilities target wheel package is under bookworm instead of bullseye. Update the README to make it consistent with the current build behavior.

* README.md: update build instrucrions

SuvarnaMeenakshi committed:
Fix to use IPv6 linklocal address as snmp agent address (#3215)

What I did
If link local IPv6 address is added as SNMP agent address, it will fail.
This PR requires changes in snmpd.conf.j2 template here sonic-net/sonic-buildimage#18350

How I did it
Append scope id to ipv6 link local IP address.

How to verify it
Able to configure link local ipv6 address as snmp agent address
sudo config snmpagentaddress add fe80::a%eth0

ayurkiv-nvda committed:
[Mellanox] Add support for Mellanox-SN4700-O32 and Mellanox-SN4700-V64 (#3450)

Signed-off-by: Andriy Yurkiv <ayurkiv@nvidia.com>

vivekrnv committed:
Fix kexec_unload failure on secure boot enabled platforms (#3439)

noaOrMlnx committed:
fix typo of neighbors header
